templates/python/job/tests/test_main.py

```diff
@@ -0,0 +1,6 @@
+import unittest
+
+
+class TestMain(unittest.TestCase):
```

Comment:
I'd include something that uses pytest since it's going to be available in the environment anyways.